infrastructure/lambda/template.yaml

```diff
@@ -80,6 +80,14 @@ Mappings:
       integration: vpce-02e2c1e10f0be4fdd
       production: vpce-02c08839b3c7c33eb
 
+  IIQAPIDatabaseModeMapping:
+    Environment:
+      dev: Aged
+      build: Aged
```

Comment:
we dont need to use aged database for build or dev, we could use static completely.

A change to the stub will be needed to return the DB only and not the aged DB. 

https://github.com/alphagov/di-ipv-stubs/pull/126